src/main/java/org/apache/commons/compress/utils/BoundedArchiveInputStream.java

```diff
@@ -0,0 +1,98 @@
+/*
```

Comment:
Is it time to depend on Commons IO instead of creating a mini Commons IO in Compress?

Reply:
I'm +1 for this.
But IIRC there was a issue or mailing discussing about this : the `skip` implemention in Commons IO didn't use the `skip` of the InputStream it's reading from. I will try to find this.